### PR TITLE
feat: Add public dataset GazeOnFaces

### PR DESCRIPTION
**Pull request by prassepaul**
## Description

added GazeOnFaces dataset

Fixes issue #

## Implemented changes

Insert a description of the changes implemented in the pull request.

- [x] Added a new dataset: GazeOnFaces

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by SiQube — APPROVED** [review]

File: src/pymovements/datasets/__init__.py [hunk 1]
```diff
@@ -50,6 +52,7 @@
     'GazeBase',
     'GazeBaseVR',
     'JuDo1000',
+    'GazeOnFaces',
```
Comment (dkrako) [inline src/pymovements/datasets/__init__.py:55]: please sort in alphabetically

**Review by dkrako — CHANGES_REQUESTED** [review]
Thanks a lot! 

please also add the two relevant test cases to https://github.com/aeye-lab/pymovements/blob/main/tests/datasets/datasets_test.py

File: src/pymovements/datasets/__init__.py (hunk 1 above)
Comment (prassepaul) [reply src/pymovements/datasets/__init__.py:54]: k

**Review by dkrako — APPROVED** [review]
Great! Thanks a lot!

**Pull request merged**